src/fetch.ts

```diff
@@ -146,6 +145,7 @@ function parseResponse(fetchRes: Response, body: Record<string, Block>): Block {
  * @param options.rpcUrl - The URL to send the request to.
  * @param options.originHttpHeaderKey - If provider, the origin field for each JSON-RPC request
  * will be attached to each outgoing fetch request under this header.
+ * @param options.req
```

Comment:
Nit: Another instance — what do you think about adding this back when the JSDoc rules are re-enabled?